managing_conflicts/src/components/CardsGrid/CardsGrid.js

```diff
@@ -16,25 +16,28 @@ const cardsGrid = (props) => {
                 }
             })
             return <Card
-                key={brochure.id}
+                id={brochure.id}
                 type="brochure"
                 name={brochure.name}
-                body={scenarios}/>
+                body={scenarios}
+                history={props.history}
+            />
         })
     } else {
         cards = props.scenarios.map(scenario => {
             return <Card
-                key={scenario.id}
```

Comment:
Aici ar trebui sa ramana key. Warning: Each child in a list should have a unique "key" prop. --> warning din consola